news/tests.py

```diff
@@ -214,6 +220,15 @@ def toggle(pk, attr):
         self.assertEquals(toggle(self.article.pk, 'hidden'), {'color': 'grey' if hidden else 'yellow'})
         self.assertEquals(toggle(self.article.pk, 'hidden'), {'color': 'yellow' if hidden else 'grey'})
 
+    def test_ticketholder(self):
+        self.add_permission("change_event")
+        self.ticketholders[0].active = False
+        expected_string = ','.join([ticketholder.email for ticketholder in self.ticketholders if ticketholder.active])
+        response = self.client.get(reverse('timeplace-tickets', args=[self.timeplace.pk]))
```

Comment:
For some reason this returns 404, and I don't know why

Reply:
I found the problem. Since I had not defined `number_of_tickets` in the timeplace, it just returned 404.